### DISCUSSION TITLE
關於Deepfake

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#618] [科技新知]
https://www.storm.mg/lifestyle/4009959?page=1
前陣子有小玉濫用AI換臉犯法之行為，近期也有中國依龍馬，原本以為是真人，但後來也才發現是仰賴於AI換臉技術的福，但此項技術難道只有這種用途嗎?深入了解這技術，才知道這叫做[Deep fake]，中文為[深偽]，在醫學用途上甚至能幫助聾啞人獲得聲音。

**Comment by 11timyyy** [comment]
拿來幫助別人，而非做出傷風敗俗的事，才應該是這項技術的目的!

**Comment by leon10456** [comment]
我還以為那是真的 原來是假的 感謝分享

**Comment by HsuChen88** [comment]
讓聾啞人有聲音好像也只能透過影片的方式，而非現實生活恢復發聲？

**Comment by awkward-willy** [comment]
這樣技術雖然是一大突破但也衍伸出很多不同的問題
包括各式各樣的犯罪案件
可見技術在發展之餘，人們的道德跟價值觀也得跟上才行

**Comment by yenwei12138** [comment]
太多原本要用來幫助社會進步的科技被不肖人士拿來幹壞事的例子了，大家真的要具備科技倫理，別做傷天害理的事，也時時注意個人隱私

**Comment by jason79461385** [comment]
但我看完文章後，只記得 I'm一籠馬 chinese一籠馬四可  艘兩圈

**Comment by sys511613** [comment]
這東西用在醫療、藝術、娛樂領域確實是很棒。但用在不對的地方甚至造成別人困擾就不好了。

**Comment by alexthebigpp** [comment]
deepfake是一項很厲害的技術，但感覺也很容易被人拿來做壞事。

**Comment by Ben6uo** [comment]
deepfake 是一個專用來惡搞別人的 但同時可能被人用來做一些不好的東西

**Comment by etin2468** [comment]
只能說每項技術被不同的人拿到就會有不同的效果...

**Comment by Lynn12378** [comment]
科技本身就是一種雙面刃，如果沒有法律嚴格限制，很容易被濫用
